components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -574,46 +562,13 @@ std::string BraveReferralsService::BuildReferralFinalizationCheckPayload()
 }
 
 void BraveReferralsService::FetchReferralHeaders() {
```

Comment:
nit: How about renaming to `SetReferralHeaders()` or something like as it doesn't fetch at all from network?

Reply:
fixed